Makefile

```diff
@@ -86,6 +87,16 @@ else
 			 cd vendor/github.com/celo-org/bls-zexe/bls && cargo build --release --target=x86_64-linux-android --lib
 endif
 
+bls-zexe-ios:
+ifeq ("$(RUSTUP_exists)","")
+	$(error "No rustup in PATH, consult https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md")
+else ifeq ("$(CARGO_LIPO_exists)","")
+	$(error "No cargo lipo in PATH, run cargo install cargo-lipo")
+else
+	rustup target add aarch64-apple-ios armv7-apple-ios x86_64-apple-ios i386-apple-ios
+	cd vendor/github.com/celo-org/bls-zexe/bls && cargo lipo --release --targets=aarch64-apple-ios,armv7-apple-ios,x86_64-apple-ios,i386-apple-ios
+endif
```

Comment:
Isn't there an endif missing here?

Reply:
Good question 👍  this syntax works but someone mentions it's failing with GNU Make 3.80:
https://stackoverflow.com/a/12297193/158525

My machine has GNU Make 3.81 which was first released in 2006.
I think we're safe, but let me know if you still wanna change it.

Reply:
Ah I just changed the structure anyway so no questions about this anymore :D